docker-compose.yml

```diff
@@ -26,6 +26,7 @@ services:
         - "api_postgres_db"
       
     auth_service:
+      command: "/src/clients.yml"
```

Comment:
This passes in the argument "/src/clients.yml" to the docker container. 🤷‍♂️ not sure why it wasn't working for me before